wowchemy-cms/templates/single.html

```diff
@@ -11,5 +11,26 @@
 <body>
 <!-- Include the latest Netlify CMS v2.x.x script that builds the admin panel. -->
 <script src="https://cdn.jsdelivr.net/npm/netlify-cms@2/dist/netlify-cms.min.js"></script>
+{{ $shortcodes := resources.Get "js/shortcodes-cms.js" }}
+<script src="{{ $shortcodes.RelPermalink }}"></script>
+{{ $post_template := resources.Get "js/cms-preview-templates/post.js" }}
+<script src="{{ $post_template.RelPermalink }}"></script>
+{{ $custom_preview_css := resources.Get "css/custom-preview.css" }}
+<script>
+  // Register any CSS file on the home page as a preview style
+  fetch("/")
```

Comment:
Fetches all stylesheets used on the site homepage (including Wowchemy.css, Google Fonts, etc.) and loads them into the CMS preview.

Reply:
@paazca doesn't `fetch("/")` return the root domain of the URL, which is not necessarily the URL of the Hugo site (Hugo's baseURL)? E.g. a site Hugo site at `an-example-host.com/my-site/` shouldn't be grabbing assets from an-example-host.com`...

Reply:
@gcushen I've now changed it to fetch the BaseURL.